projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -66,11 +66,21 @@ ${ repoLabels
 	.join( '\n' ) }
 
 Analyze the issue and suggest relevant labels. Rules:
+# Updated prompt: Ensures 'Product', 'Platform' and 'Type' issues are always added.
 - Use only existing labels provided.
+- Include 1 '[Product]' label.
 - Include 1 '[Feature Group]' label.
 - Include 1 to 3 '[Feature]' labels.
+- Include the "[Platform] Simple" AND/OR "[Platform] Atomic" labels as appropriate.
+- Only if NOT present: Add 1 of the following labels based on your assessment of the issue type: '[Type] Bug', '[Type] Enhancement', or '[Type] Feature'.
```

Comment:
Same here, neither "Platform" nor "Type" are included in the list above.

Reply:
Also, I wonder whether the AI will have any clue about what either of the platforms are and when they might apply.